plugins/ltac/g_auto.mlg

```diff
@@ -61,12 +61,17 @@ let eval_uconstrs ist cs =
   let map c env sigma = c env sigma in
   List.map (fun c -> map (Tacinterp.type_uconstr ~flags ist c)) cs
 
+
+let gcc () = let env = Global.env () in Evd.from_env env, env
+
 let pr_auto_using_raw _ _ _  = Pptactic.pr_auto_using Ppconstr.pr_constr_expr
+
 let pr_auto_using_glob _ _ _ = Pptactic.pr_auto_using (fun (c,_) ->
-    let _, env = Pfedit.get_current_context () in
```

Comment:
here the env is wrong, see above for an explanation

Reply:
> here the env is wrong, see above for an explanation

Where's "above" @gares ? Maybe it got lost on a rebase. Indeed here we are not using the best ENV, however I am not sure it is worth to change all the extension point printing interface.

Regardless of that, that'd be a minor change to make.

Reply:
> here the env is wrong, see above for an explanation

I don't know how but a comment of mine disappeared.

The `env` data type is used for two things: the global environment and the proof context. The named_context in there contains the section variables + the proof variables.

In the code above you pick the global one (no proof variables) while the old code was taking the proof environment (of the first goal).

Reply:
Yup we will fix that and let the extension point send an environment. (Or we could use the compat layer)